proto/zitadel/user/v2alpha/auth.proto

```diff
@@ -20,7 +20,7 @@ message SendPasskeyRegistrationLink {
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
       min_length: 1;
       max_length: 200;
-      example: "\"https://example.com/passkey/register?userID={{.UserID}}&code={{.Code}}&orgID={{.OrgID}}\"";
+      example: "\"https://example.com/passkey/register?userID={{.UserID}}&orgID={{.ResourceOwner}}&codeID={{.CodeID}}&code={{.Code}}\"";
```

Comment:
depending on whether we rename the field to OrgID, we would need to change here as well